test/sagemaker_tests/tensorflow/tensorflow2_training/integration/local/test_horovod.py

```diff
@@ -30,6 +30,8 @@
 def test_distributed_training_horovod_basic_singlenode(
     instances, processes, sagemaker_local_session, docker_image, tmpdir, framework_version
 ):
+    if framework_version >= "2.13":
```

Comment:
framework version needs to be handled here and all other occurrences in this PR

Reply:
Do not understand what you mean here as this statement covers the expected version, and above, doing exactly what is needed. What needs to be handled here?

Reply:
using a completely different process. Thank you for the assist on this.

Reply:
@tejaschumbalkar Do you mean that this should use `Version(framework_version) in SpecifierSet(">=2.13")`?

Reply:
as @saimidu was showing, this was created in a fixture. more work will be done on the SM PR right after this one.